experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -236,21 +236,16 @@ function empty_covered_projective_scheme(R::T) where {T<:AbstractAlgebra.Ring}
   return CoveredProjectiveScheme(Y, C, pp, tr)
 end
 
-@Markdown.doc """
-    blow_up(W::AbsSpec, I::Ideal)
 
-Return the blowup of ``W`` at the ideal ``I``; this is a `ProjectiveScheme` 
-with `base_scheme` ``W``.
-"""
-function blow_up(W::AbsSpec, I::Ideal; var_name::String="s")
-  error("method `blow_up` not implemented for arguments of type $(typeof(W)) and $(typeof(I))")
+function blow_up_chart(W::AbsSpec, I::Ideal; var_name::String="s")
```

Comment:
Even though it is internal please document it. Not for the user but for us.

Reply:
documentation added